src/webui/src/components/trial-detail/Para.tsx

```diff
@@ -571,89 +591,83 @@ class Para extends React.Component<ParaProps, ParaState> {
         }
     }
 
-    shouldComponentUpdate(nextProps: ParaProps, nextState: ParaState): boolean {
+    // shouldComponentUpdate(nextProps: ParaProps, nextState: ParaState): boolean {
 
-        const { whichGraph } = nextProps;
-        const beforeGraph = this.props.whichGraph;
-        if (whichGraph === '2') {
-            if (whichGraph !== beforeGraph) {
-                return true;
-            }
+    //     const { whichGraph } = nextProps;
```

Comment:
If these lines are not useful, could remove them.

Reply:
Ok.